internal/pkg/exchange-scrapers/CoinBaseScraper.go

```diff
@@ -134,6 +137,31 @@ func (s *CoinBaseScraper) Close() error {
 	return s.error
 }
 
+// FetchAvailablePairs returns a list with all available trade pairs
+func (s *CoinBaseScraper) FetchAvailablePairs() (pairs []dia.Pair, err error) {
+	response, err := http.Get("https://api.pro.coinbase.com/products")
+	if err != nil {
+		log.Error("The HTTP request failed:", err)
+		return
+	}
+	defer response.Body.Close()
+	data, _ := ioutil.ReadAll(response.Body)
+	var ar []gdax.Product
+	err = json.Unmarshal(data, &ar)
+	if err == nil {
+		pairs = make([]dia.Pair, len(ar))
+		log.Print("Retrieved:", len(ar))
+		for i, p := range ar {
+			pairs[i] = dia.Pair{
+				Symbol:      p.BaseCurrency,
+				ForeignName: p.Id,
+				Exchange:    dia.CoinBaseExchange,
```

Comment:
maybe would be good occasion to get rid of this constant, like by passing the info in the NewScrapper ?